geodepy/tests/resources/Test_Conversion_ISG_Geo.csv

```diff
@@ -0,0 +1,212 @@
+ALBU,-36.079078850050,146.914301020650,
```

Comment:
Do we need this many test points? Perhaps, a few from each zone would be enough?

Reply:
Happy to reduce the set, I was just mirroring the existing files as much as possible, so I used all the CORS in NSW for simplicity. I could reduce it to 5 per zone? Which would be around 50 marks - might not have 5 marks for 572 (Lord Howe Island).

Reply:
Gotcha! OK, no, leave it as it is for now. I'm going to merge the changes. Thanks for your work on this